BDArmory/Distribution/GameData/BDArmory/ChangeLog.txt

```diff
@@ -9,17 +9,19 @@
 	- Fix the wrongly labeled "ru-ru" localisation to "ru" so that it loads properly.
 	- Add additional Chinese localisation thanks to user ZIYUXUAN201011.
 	- Updated German localisation from EzBro.
+	- UIs now scale with KSP UI scale.
```

Comment:
This is already in v1.8.0.0, unless you're meaning something different?
> - Added an option to follow the stock UI scaling instead of BDA's one.

Reply:
That's an oops, then; read the Git history too far back